scripts/ci/node-template-release/Cargo.lock

```diff
@@ -0,0 +1,578 @@
+# This file is automatically @generated by Cargo.
```

Comment:
https://github.com/paritytech/substrate/tree/master/.maintain/node-template-release does not include a Cargo.lock, perhaps it is on purpose?

Reply:
Yeah this file should be removed.

Reply:
Removed.